src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__content {
+    display: block;
+    padding: 0 16px 16px;
+  }
+
+  &__title {
+    font-size: 12px;
+    text-align: left;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    text-align: left;
+    margin: 0;
+    padding-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 16px;
+    line-height: 18px;
+  }
+
+  &__price-label {
+    color: $secondary-color;
+    margin: 0;
+    font-size: 12px;
+  }
+
+  &__price-value {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    border-radius: 5px;
+    background: $blue-accent;
+    padding: 12px;
+
+    color: $white-color;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+
+  &__button:hover {
+    border: 1px solid $blue-accent;
+    background-color: $white-color;
+    color: $blue-accent;
+  }
+}
+
+* {
```

Comment:
move to index.scss

Reply:
<img width="632" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/431c907c-3268-439e-8ae9-1d5960db2b5e">